ball_detection.py

```diff
@@ -1,43 +1,85 @@
 import cv2
 import numpy as np
-import imutils
+
+# known diamter of ball from 90in way (not accounting for angle)
+KNOWN_DIAMETER_PX = 156  # px
+# known distance from cam to closest ball (7.5', not accounting for angle)
+KNOWN_DISTANCE = 90  # in
+# known diameter of ball
+KNOWN_DIAMETER_IN = 7  # in
+# focal len = 90 * 156 / 7
+FOCAL_LENGTH = 2005.714286  # px
 
 RADIUS_THRESH = 25  # modify
 KNOWN_RADIUS = 3.5  # in
+RADIUS_THRESH = 10  # modify
 
 
-def detectBall(frame):
-    frame = cv2.GaussianBlur(frame, (5, 5), 0)
-    hsv_frame = cv2.cvtColor(frame, cv2.COLOR_BGR2HSV)
-    lower_yellow = np.array([22, 28, 82])  # values from field image #fix this range
+def distance(flength, kwidth, pwidth):
+    dist = ((kwidth * flength) / pwidth) / 12
+    print("DISTANCE: ", dist)
+    return round(dist, 1)
+
+
+def detectBall(frame, w, h):
```

Comment:
Do we need to pass in w,h? You already have frame, so you can just do frame.shape[0] or frame.shape[1] when you need to.